test/libsolidity/util/BytesUtils.cpp

```diff
@@ -96,6 +96,25 @@ bytes BytesUtils::convertNumber(string const& _literal)
 	}
 }
 
+bytes BytesUtils::convertFixedPoint(string const& _literal, size_t *_fixedPointN)
```

Comment:
Please avoid returning by pointer.

Reply:
Also please document.

Reply:
Or maybe name this `parseFixedPointLiteral`?

Reply:
Ah, these are all called `convert...` and it is documented in the header - sorry.